MockKit/MockCGMManager.swift

```diff
@@ -249,12 +341,18 @@ public final class MockCGMManager: TestingCGMManager {
             let currentValue = samples.first
         {
             switch currentValue.quantity.doubleValue(for: HKUnit.milligramsPerDeciliter) {
-            case ..<mockSensorState.lowGlucoseThreshold.doubleValue(for: HKUnit.milligramsPerDeciliter):
+            case ..<mockSensorState.cgmLowerLimit.doubleValue(for: HKUnit.milligramsPerDeciliter):
```

Comment:
[nit] From above recommendation to include a "units" property on the state, use that here as well to remove hard-coded mg/dL.

Reply:
not sure why I didn't do this originally, but one can compare `HKQuantity`s.